clang/unittests/Format/FormatTest.cpp

```diff
@@ -10539,6 +10539,17 @@ TEST_F(FormatTest, KeepStringLabelValuePairsOnALine) {
       "                  bbbbbbbbbbbbbbbbbbbbbbb);");
 }
 
+TEST_F(FormatTest, WrapBeforeInsertionOperatorbetweenStringLiterals) {
+  verifyFormat("QStringList() << \"foo\" << \"bar\";");
```

Comment:
Will `verifyNoChange` here be better suitable?

Reply:
No, because the single-argument version of `verifyFormat` is a more stringent test than `verifyNoChange` as the former also tests the input with all optional whitespace characters between tokens removed.